### ISSUE TITLE
Stacked Graph does not work

### ISSUE THREAD

**Issue by deathowl** [fdietz/team_dashboard#64]
if i select "stacked graph " in graph settings, the result is a simple line graph on the canvas.

**Comment by fdietz** [comment]
Hi Csergo,

this is a known problem, since we used the rickshaw library which supported
stacked graphs but had to switch to flotr2 due to rendering issues. I hope
to switch back to rickshaw (the developers fixed our bug recently) and then
we have this functionality back again.

I keep you posted about the progress in this issue!

Cheers
Frederik

**Comment by deathowl** [comment]
may i ping this thread? Is it fixed already?

**Comment by fdietz** [comment]
Hi Csergo,

I've changed strategies since the graphing library doesn't support it or
has missing features. After the version 2 release, I will work on
integrating native support for Graphite by rendering the graphite output as
is. That way we can easily support all graphite features and won't have any
bugs due to wrong mappings between other graph libraries and graphite-web
API.

I've created an issue to track it here:
https://github.com/fdietz/team_dashboard/issues/80

Cheers,
Frederik
